src/app/auth/components/registration/registration.component.html

```diff
@@ -0,0 +1,66 @@
+<div>
+  <h5>Введите данные для регистрации</h5>
+  <h6>Поля обязательные для заполнения (<span class="color-red">*</span>)</h6>
+  <form [formGroup]="form" (ngSubmit)="onSubmit()">
+    <div>
+      <div class="form-input">
+        <span [ngClass]="form.get('email').invalid && form.get('email').touched ? 'color-red' : ''">*</span>
+        <app-input [formControlName]="inputFields.email"
+                   [name]="inputFields.email"
+                   [placeholder]="inputFields.email"
+        >
+        </app-input>
+      </div>
+    </div>
+    <div class="form-group">
+      <div class="form-input">
+        <span [ngClass]="form.get('password').invalid && form.get('password').touched ? 'color-red' : ''">*</span>
+        <app-input [type]="inputFields.password"
+                   [placeholder]="inputFields.password"
+                   [formControlName]="inputFields.password"
+                   [name]="inputFields.password"
+        >
+        </app-input>
+      </div>
+    </div>
+    <div>
+      <div class="form-input">
+        <span [ngClass]="form.get('name').invalid && form.get('name').touched ? 'color-red' : ''">*</span>
```

Comment:
Сделать везде по такому типу
```suggestion
        <span [class.color-red]="form.get('name').invalid && form.get('name').touched>*</span>
```